llvm/include/llvm/ADT/SetOperations.h

```diff
@@ -157,6 +157,23 @@ bool set_is_subset(const S1Ty &S1, const S2Ty &S2) {
   return true;
 }
 
+template <class S1Ty, class S2Ty>
+bool set_intersects_impl(const S1Ty &S1, const S2Ty &S2) {
```

Comment:
Consider putting this in a `detail` namespace

Reply:
done